Common/Product/SharedProject/CommonPackage.cs

```diff
@@ -214,9 +214,11 @@ public abstract class CommonPackage : Package, IOleComponent {
         }
 
         internal static void OpenWebBrowser(string url) {
-            var uri = new Uri(url);
-            Process.Start(new ProcessStartInfo(uri.AbsoluteUri));
-            return;
+            try {
+                var uri = new Uri(url);
+                Process.Start(new ProcessStartInfo(uri.AbsoluteUri));
+            } catch (UriFormatException) {
+            }
```

Comment:
Can we report the error? Since this should only occur due to corruption on the user's machine, it seems like a reasonable thing to pop up in a dialog box.